testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -440,9 +493,18 @@ public void shouldUsePaginationForSimpleStatement()
             Thread.sleep(100);
         }
 
-        assertThat(false).isTrue();
+        fail("Time elapsed waiting for test condition");
         return null;
     }
 
+    private static ProgrammaticDriverConfigLoaderBuilder getDriverConfigLoaderBuilder()
+    {
+        return DriverConfigLoader.programmaticBuilder()
+            .withBoolean(DefaultDriverOption.METADATA_TOKEN_MAP_ENABLED, false)
+            .withString(DefaultDriverOption.LOAD_BALANCING_POLICY_CLASS, DcInferringLoadBalancingPolicy.class.getName())
+            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(5))
+            .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(5))
```

Comment:
if you upgraded to java-driver `4.9`, you don't need to override this setting any more - the default is `5 seconds`.

Reply:
Removed it.